demo/registry.yaml

```diff
@@ -50,3 +43,11 @@ chains:
     faucet: http://49.12.73.189:8002
     rpc:
       - http://188.34.162.78:26657
+  cosmoshub-4:
+    chain_id: cosmoshub-4
+    prefix: cosmos
+    gas_price: 0.01uatom
```

Comment:
Are you sure this gas price works?
I have used `0.025uatom` as a default for some time. Happy if it is cheaper now.

Reply:
Yeah I made it `0.026uatom` after some tx timeouts. I was not sure what was the default uatom gas price among cosmos hub validators.